packages/typespec-client-generator-core/test/decorators.test.ts

```diff
@@ -3688,4 +3688,40 @@ describe("typespec-client-generator-core: decorators", () => {
       ok(clients[0].type);
     });
   });
+
+  describe("@overrideClientMethod", () => {
+    it("basic", async () => {
+      await runner.compileWithCustomization(
+        `
+        @service
+        namespace MyService;
+        model Params {
+          foo: string;
+          bar: string;
+        }
+
+        op original(...Params): void;
+        `,
+        `
+        namespace MyCustomizations;
+        model NewModel {
+          params: MyService.Params;
+        }
+        op customization(...NewModel): void;
```

Comment:
@tadelesh 

Would this `op customization` be picked by TCGC as client method?

Reply:
it isn't because it's not under a client